documentation/endpoints/notifications.md

```diff
@@ -0,0 +1,17 @@
+# Notifications
+
+## Post relative reminder
+
+```
+POST /v1/notifications/reminder
+```
+
+Returns either an `error` or `success` object.
+
+**Parameters**
+
+Name | Type | Description
+--- | --- | ---
+`northstarUserId` | `string` | Northstar User Id to send notification for.
+`campaignId` | `string::int` | The campaign ID this notification is for.
```

Comment:
I have no idea what format this is, traditionally I would do `|` but that breaks the markdown table.